AlphaWallet/TokenScriptClient/Models/TokenScriptSupportable.swift

```diff
@@ -1,36 +1,25 @@
 //
-//  TokensService.swift
+//  TokenScriptSupportable.swift
 //  AlphaWallet
 //
 //  Created by Vladyslav Shepitko on 18.05.2022.
 //
 
 import Foundation
-import BigInt
-import AlphaWalletOpenSea
+import BigInt 
 
-protocol Tokenable {
+protocol TokenScriptSupportable {
```

Comment:
This protocol doesn't seem like it's specific for TokenScript?

Reply:
it just brings abstraction for parts where token script is using, `generate xmlHandler` and so on. So we replace accepting exact token `Type` with `TokenScriptSupportable`

Reply:
OK